src/collapse/scss/variable.scss

```diff
@@ -77,6 +77,12 @@ $collapse-icon-margin-r: $s-2 !default;
 /// margin (l)
 /// @namespace size/icon
 $collapse-icon-margin-l: $s-3 !default;
+/// collapsed rotation
+$collapse-icon-rotation-collapsed: 90deg;
+/// expanded rotation
+$collapse-icon-rotation-expanded: 180deg;
+/// collapsed rotation rtl
+$collapse-icon-rotation-collapsed-rtl: $collapse-icon-rotation-collapsed * 3;
```

Comment:
use double slashes( // ) for normal comment
treble slashes( /// ) are used to note meta data for theme configuration platform

Reply:
these variables are not for theme configuration platform.
you may not need to add comment or add comment with double slashes( // )

Reply:
I have fixed, please review